content/en/docs/concepts/what-is-opentelemetry.md

```diff
@@ -3,34 +3,42 @@ title: "What is OpenTelemetry?"
 weight: 10
 ---
 
-OpenTelemetry is a set of APIs, SDKs, tooling and integrations that are designed
-for the creation and management of _telemetry data_ such as traces, metrics, and
-logs. The project provides a vendor-agnostic implementation that can be
-configured to send telemetry data to the backend(s) of your choice. It supports
-a variety of popular open-source projects including Jaeger and Prometheus.
+Microservices architectures have enabled developers to build and release software faster and with greater independence, as they were no longer beholden to the elaborate release processes associated with monolithic architectures. 
 
-## Why you need OpenTelemetry and what it can do
+As these now-distributed systems scaled, it became increasingly difficult for developers to see how their own services depend on or affect other services, especially after a deployment or during an outage, where speed and accuracy are critical.
 
-In cloud-native technology stacks, distributed and polyglot architectures are
-the norm. Distributed architectures introduce a variety of operational
-challenges including how to solve availability and performance issues quickly.
-These challenges have led to the rise of observability.
+>[Observability](/docs/concepts/observability-primer/#what-is-observability) has made it possible for both developers and operators to gain that visibility into their systems.
 
-Telemetry data is needed to power observability products. Traditionally,
-telemetry data has been provided by either open-source projects or commercial
-vendors. With a lack of standardization, the net result is the lack of data
-portability and the burden on the user to maintain the instrumentation.
+## So what?
 
-The OpenTelemetry project solves these problems by providing a single,
-vendor-agnostic solution. The project has broad industry support and adoption
-from cloud providers, vendors and end users.
+In order to make a system observable, it must be instrumented. That is, the code must emit [traces](/docs/concepts/observability-primer/#distributed-traces), [metrics](/docs/concepts/observability-primer/#reliability--metrics), [logs](/docs/concepts/observability-primer/#logs). The instrumented data must then be sent to an Observability back-end. There are a number of Observability back-ends out there, ranging from self-hosted open-source tools (e.g. [Jaeger](https://www.jaegertracing.io/) and [Zipkin](https://zipkin.io/)), to commercial SAAS offerings (e.g. [Datadog](https://datadog.com), [Dynatrace](https://dynatrace.com), [Lighstep](https://lightstep.com), [Honeycomb](https://honeycomb.io), etc.).
```

Comment:
as far as I know we don't advertise commercial solutions, so take out datadog, dynatrace, lightstep, honeycomb.

Reply:
Noted - will remove!